### PR TITLE
Clean labs to be compatible with OpenELIS

### PR DESCRIPTION
**Pull request by reagan-meant**
@rbuisson Need review of the concept names

### CODE REVIEW

**Review by rbuisson — APPROVED** [review]

File: configuration/concepts/lab_tests.csv [hunk 1]
```diff
@@ -1,12 +1,10 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:800
 718b4589-2a11-4355-b8dc-aa668a93e098,,CIEL:664,Negative,Négatif,Negative,Négatif,,,Misc,N/A,,,,,,,,,,,
 7acfafa4-f19b-485e-97a7-c9e002dbe37a,,CIEL:703,Positive,Positif,Positive,Positif,,,Misc,N/A,,,,,,,,,,,
+11f699e3-6983-4b27-99cd-07d4bd323a2f,,CIEL:163748,Present,Instant,Present,Instant,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Would be "Présent" in fr.

File: configuration/concepts/lab_tests.csv [hunk 2]
```diff
@@ -77,53 +91,24 @@ d00b8f8d-99f1-4d1b-806b-f2ca1353f7f8,,,Stool Consistence,Selles consistance,Stoo
 6f657810-f33d-4ff6-80f3-d98bf78934f6,,CIEL:133792,Mucus,Glaireuse,Mucus,Glaireuse,,,Misc,N/A,,,,,,,,,,,
 5638d070-ab46-4fde-9e37-90b1fc73ac3b,,,Sanguineous,Sanguignolente,Sanguineous,Sanguignolente,,,Misc,N/A,,,,,,,,,,,
 490162ab-7c7d-4574-a43a-274df603b741,,,Mucosanguineous,Glairosanguignolente,Mucosanguineous,Glairosanguignolente,,,Misc,N/A,,,,,,,,,,,
-40f3ff80-44ff-4fad-b0eb-fa89ff05e1f8,,CIEL:163640,Stool Appearance,Selles aspect,Stool Appearance,Selles aspect,,,LabTest,Coded,,Mucus;Sanguineous;Mucosanguineous;Normal,,,,,,,,,
+40f3ff80-44ff-4fad-b0eb-fa89ff05e1f8,,CIEL:163640,Stool Appearance,Selles aspect,Stool Appearance,Selles aspect,,,LabTest,Coded,,Mucus;Sanguineous;Mucosanguineous,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Would be "Aspect des selles" in fr

File: configuration/concepts/lab_tests.csv [hunk 3]
```diff
@@ -77,53 +91,24 @@ d00b8f8d-99f1-4d1b-806b-f2ca1353f7f8,,,Stool Consistence,Selles consistance,Stoo
...
 490162ab-7c7d-4574-a43a-274df603b741,,,Mucosanguineous,Glairosanguignolente,Mucosanguineous,Glairosanguignolente,,,Misc,N/A,,,,,,,,,,,
-40f3ff80-44ff-4fad-b0eb-fa89ff05e1f8,,CIEL:163640,Stool Appearance,Selles aspect,Stool Appearance,Selles aspect,,,LabTest,Coded,,Mucus;Sanguineous;Mucosanguineous;Normal,,,,,,,,,
+40f3ff80-44ff-4fad-b0eb-fa89ff05e1f8,,CIEL:163640,Stool Appearance,Selles aspect,Stool Appearance,Selles aspect,,,LabTest,Coded,,Mucus;Sanguineous;Mucosanguineous,,,,,,,,,
 2093e612-afcc-45bf-a4ed-db18db204a48,,,Greenish,Verdâtre,Greenish,Verdâtre,,,Misc,N/A,,,,,,,,,,,
 e901a01d-8297-43f4-b39d-05e5ce09b708,,,Blackish,Noirâtre,Blackish,Noirâtre,,,Misc,N/A,,,,,,,,,,,
 196e8b35-0f53-4286-be75-65a945056a0d,,,Yellowish,Jaunâtre,Yellowish,Jaunâtre,,,Misc,N/A,,,,,,,,,,,
 90a106a3-02f6-4412-bde9-8d8481ea803a,,,Reddish,Rougeâtre,Reddish,Rougeâtre,,,Misc,N/A,,,,,,,,,,,
 43084e74-2ee3-438b-81c8-57496c5f286a,,,Whitish,Blanchâtre,Whitish,Blanchâtre,,,Misc,N/A,,,,,,,,,,,
 88094e41-e2c7-443f-8770-f46c78e03e7e,,CIEL:163641,Stool Colour,Selles Couleur,Stool Colour,Selles Couleur,,,LabTest,Coded,,Greenish;Blackish;Yellowish;Reddish;Whitish,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Would be "Couleur des selles"

**Review by rbuisson — CHANGES_REQUESTED** [review]
Just few translation changes

File: configuration/concepts/lab_tests.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv [hunk 4]
```diff
@@ -53,22 +71,18 @@ e653561e-861c-4406-a576-270518b4d2e4,,,Hyaline Cylinders,Cylindres hyalins,Hyali
...
-e03a11de-68f8-4995-b004-72c66405f458,,,Two Plus Parasite,2+,2+,2+,,,Misc,N/A,,,,,,,,,,,
-e8a5e2dd-8444-4d8e-bbda-1d862743ffed,,,Three Plus Parasite,3+,3+,3+,,,Misc,N/A,,,,,,,,,,,
-dbaf6d63-723b-4399-9af1-d8f220f66fd5,,,Four Plus Parasite,4+,4+,4+,,,Misc,N/A,,,,,,,,,,,
-6c112abd-3f21-440a-83aa-6ae7de311ecf,,,Malaria Density,Malaria desité,Malaria Density,Malaria desité,,,LabTest,Coded,,One Plus Parasite;Two Plus Parasite;Three Plus Parasite;Four Plus Parasite,,,,,,,,,
+c108ee98-5abc-43f3-9653-fc920d79e133,,,Urinalysis,Urinalysis,Urinalysis,Urinalysis,,,LabSet,N/A,,,Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Albumin;Ascorbic Acid;Bilirubin;Urobilinogen;Protein;Leucocytes;White Blood Cells;Red Blood Cells;Bacteria;Epithelial Cells;Budding Yeasts;Simple Yeasts;Mycelium;Macrophages;Kidney Cells;Leukocyte Cylinders;Granular Cylinders;Hyaline Cylinders;Erythrocyte Cylinders;Crystals;Trichomonas Vaginalis,,,,,,,,
+909a373e-b077-413d-a43e-dc4461f51f20,,CIEL:159663,Gram positive cocci,Coccies Gram +,Gram + cocci,Coccies Gram +,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: En: "Gram-positive Cocci" / "Gram+ Cocci"
Fr: "Cocci à Gram Positif" / "Cocci Gram+"

File: configuration/concepts/lab_tests.csv [hunk 5]
```diff
@@ -36,12 +55,11 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
...
 a360c76e-b365-4854-97f9-1703ec29cb48,,,Protein,Protéines,Protein,Protéines,,,LabTest,Text,,,,,,,,,,,
-245ff935-a1f7-4dc6-8a1d-f2ff62ee2af9,,,Urine Macroscopy,Urine macroscopie,Urine Macroscopy,Urine macroscopie,,,LabSet,N/A,,,Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Albumin;Ascorbic Acid;Bilirubin;Urobilinogen;Protein,,,,,,,,
 e5dc4fb9-5a3c-4956-9a39-66aaaf64adcc,,,Leucocytes,Leucocytes,Leucocytes,Leucocytes,,,LabTest,Text,,,,,,,,,,,
 e3ae7371-8a91-413e-8f99-93bc8f45a07a,,,White Blood Cells,Globules blancs,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
 668b296c-a1fc-4d2f-bfad-1997704a4762,,,Red Blood Cells,Globules rouges,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
 168f4adf-4beb-49d2-8070-b2776ccd2b23,,,Bacteria,Bactéries,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,
-1d832608-72a0-4e18-af6c-5d3bc2c18caf,,,Epithelail Cells,Cellules épithélailes,Epithelail Cells,Cellules épithélailes,,,LabTest,Text,,,,,,,,,,,
+1d832608-72a0-4e18-af6c-5d3bc2c18caf,,,Epithelial Cells,Cellules épithélailes,Epithelial Cells,Cellules épithélailes,,,LabTest,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: "épithéliales"

File: configuration/concepts/lab_tests.csv [hunk 6]
```diff
@@ -53,22 +71,18 @@ e653561e-861c-4406-a576-270518b4d2e4,,,Hyaline Cylinders,Cylindres hyalins,Hyali
...
+c108ee98-5abc-43f3-9653-fc920d79e133,,,Urinalysis,Urinalysis,Urinalysis,Urinalysis,,,LabSet,N/A,,,Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Albumin;Ascorbic Acid;Bilirubin;Urobilinogen;Protein;Leucocytes;White Blood Cells;Red Blood Cells;Bacteria;Epithelial Cells;Budding Yeasts;Simple Yeasts;Mycelium;Macrophages;Kidney Cells;Leukocyte Cylinders;Granular Cylinders;Hyaline Cylinders;Erythrocyte Cylinders;Crystals;Trichomonas Vaginalis,,,,,,,,
+909a373e-b077-413d-a43e-dc4461f51f20,,CIEL:159663,Gram positive cocci,Coccies Gram +,Gram + cocci,Coccies Gram +,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+19cde42f-3040-4d44-96c0-0fa740e748c8,,CIEL:159664,Gram negative cocci,Coccies Gram -,Gram - cocci,Coccies Gram -,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+485af765-a605-4438-88cc-f730414901ab,,,Gram positive diplococci,Diplocoques Gram +,Gram + diplococci,Diplocoques Gram +,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: "Gram-positive Diplococci" etc...

File: configuration/concepts/lab_tests.csv [hunk 7]
```diff
@@ -53,22 +71,18 @@ e653561e-861c-4406-a576-270518b4d2e4,,,Hyaline Cylinders,Cylindres hyalins,Hyali
...
+5d687ffe-d0d8-48fe-a9c6-ed5d47cae953,,,Gram negative intracellular diplococci,Diplocoques Gram - intracellulaire,Gram - intracellular diplococci,Diplocoques Gram - intracellulaire,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+bb645ded-3ebb-4117-9f81-51d1ce236f1f,,,Gram positive bacilli,Bacilles Gram +,Gram + bacilli,Bacilles Gram +,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+d4db3add-39c5-4d42-bd11-3dd6a32d136a,,,Gram negative bacilli,Bacilles Gram -,Gram - bacilli,Bacilles Gram -,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+1d86445f-f0af-4612-80f6-f9cacf055844,,,Coccobacilli,Coccobacilles,Coccobacilli,Coccobacilles,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+784a3266-485d-49a3-851f-44hy93df4edb,,,Vaginal Swab (Gram Stain),Frottis vaginal (Gram Stain),Vaginal Swab (Gram Stain),Frottis vaginal (Gram Stain),,,LabSet,N/A,,,Gram positive cocci;Gram negative cocci;Gram positive diplococci;Gram negative diplococci;Gram negative intracellular diplococci;Gram positive bacilli;Gram negative bacilli;Coccobacilli,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: In fr: "Frottis Vaginal (Coloration de Gram)"

File: configuration/concepts/lab_tests.csv [hunk 8]
```diff
@@ -53,22 +71,18 @@ e653561e-861c-4406-a576-270518b4d2e4,,,Hyaline Cylinders,Cylindres hyalins,Hyali
...
+23b53f61-2845-4922-82a2-63e024afbcb2,,,Clue cell,Clue cell,Clue cell,Clue cell,,,LabTest,Coded,,Present;Absent;Indeterminate,,,,,,,,,
+784a3266-485d-49a3-851f-2a2c93df4edb,,,Vaginal Swab (Wet Mount),Frottis vaginal (Wet Mount),Vaginal Swab (Wet Mount),Frottis vaginal (Wet Mount),,,LabSet,N/A,,,White Blood Cells;Red Blood Cells;Bacteria;Epithelial Cells;Budding Yeasts;Simple Yeasts;Mycelium;Trichomonas vaginalis;Clue cell,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Fr: "Frottis vaginal (Goutte pendante)"

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/concepts/lab_tests.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 7 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 8 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

**Comment by rbuisson** [comment]
Thanks @reagan-meant . Merging.

**Pull request merged**